device/dhcp/servers/v4/dhcpv4serverleases.yaml

```diff
@@ -0,0 +1,91 @@
+components:
+  schemas:
+    DhcpServerV4.Pool:
+      description: >-
+        Configuration for DHCPv4 address pool for a lease.
+      type: object
+      x-status:
+        status: under_review
+      required: [name, start_address]
+      properties:
+        name:
+          x-include: ../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease.
```

Comment:
'The duration of time in seconds that is assigned to a lease' might be better provided Life Time is not something specific from specification for this attribute.